files/chef-marketplace-cookbooks/chef-marketplace/templates/default/delivery.rb.erb

```diff
@@ -0,0 +1,10 @@
+delivery_fqdn "marketplace.vm"
```

Comment:
Not so sure we should be hardcoding this here.. Should probably use the FQDN helper